### ISSUE TITLE
Can't write on the file mounted ssd 

### ISSUE THREAD

**Issue by hasujin** [gerard/ext4fuse#39]
Hello. I've tried to ext4FUSE and i successed mount (./ext4fuse /dev/sda /mnt/mountpoint)
But, Can't write on the file
Device is NAND flash memory(SSD). 

It can't support  SSD? or it just my fault?

Please help me.
thank you.

**Comment by Oxylibrium** [comment]
Apparently, EXT4Fuse has no write support and is a read-only driver.

**Comment by Asday** [comment]
Is there any good solution for sharing data between OSX and a Linux installation?  HFS+ support is abysmal in Linux, and nothing I have found so far can read and write ext4 on OSX, and FAT and NTFS are out of the question.

**Comment by Klemart3D** [comment]
I tried to write on mounted ext4 volume with root privilege on macOS Sierra 10.13.3 but still read only.
Solution I have found is to install a Linux distribution on VirtualBox, unmount (but not eject) the drive from mac to mount it on Linux and write in it in CLI root privileges.

**Comment by GwynethLlewelyn** [comment]
Indeed, `ext4fuse` _only_ works to _read_ ext4 file partitions and doesn't even have the _capability_ to do more than that (unless, well, you patch it). It's not just bad permissions or mounting files under the 'wrong' user.  No, it's really that there is _no_ code to support any of the writing operations...
